java8/src/main/java/com/fullcontact/apilib/FullContactApi.java

```diff
@@ -18,11 +15,20 @@
 public interface FullContactApi {
 
   @POST(FCConstants.API_ENDPOINT_PERSON_ENRICH)
-  CompletableFuture<Response<PersonResponse>> personEnrich(@Body PersonRequest personRequest);
+  CompletableFuture<Response<JsonElement>> personEnrich(@Body RequestBody body);
```

Comment:
In going to `JsonElement` we're getting away from the Response, Company, etc Response objects in the models. What is your thinking here? Is this for ease of development, or does it make it easier for end users to use?

Reply:
I made the switch to `JsonElement` to simplify development and maintenance. If we keep it as `PersonResponse` or `CompanyResponse` the only advantage we get is retrofit automatically deflate to corresponding classes, but in doing so, there was a lot of duplicate code for retry logic and handling response, By changing to `JsonElement` we can use same methods, thereby reducing duplication, and deserialise using `gson` manually. For end users, there should be no difference